### DISCUSSION TITLE
Translation option

### DISCUSSION THREAD

**Discussion by throwaway898786** [AprilSylph/XKit-Rewritten#1181] [Suggestion Box]
Hello. Tumblr's language options are limited so it would be very helpful if XKit could provide more, would this option be possible? It would be nicer than needing a whole other translator extension. Sorry if this has already been suggested.

**Comment by marcustyphoon** [comment]
This indeed has not been suggested! We don't really have any better way of doing this than a generic translator extension would, though, so I don't think we'd write and maintain the huge amount of code that would require just to do something no better than something else that already exists.

**Reply by throwaway898786** [reply]
Aw. I understand, thanks so much anyway for considering. :]
